src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,21 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+    public static final double g = 9.806;
+
+    public static class Turret {
+        public static int TALON_TIMEOUT = 10;
+
+        public static int TICKS_PER_DEGREE = 0;
+
+        public static int TURRET_JOYSTICK_DEGREES = 15; //angle the turret will move in accordance to joystick position (0%-100%) every 40ms.
```

Comment:
What is the relation between the joystick position to the constants? is 100% would result 15 degrees?
also why 40ms because the roborio loop is 20ms